lib/freeclimb/models/terminate_conference.rb

```diff
@@ -29,8 +28,7 @@ def self.acceptable_attributes
 
     # Attribute type mapping.
     def self.openapi_types
-      {
-      }
+      {}
```

Comment:
do you know where all these formatting diffs are coming from?

Reply:
We've added a formatting tool to the ruby sdk which allows for both model and test files to format to a specific rule.

The package is called rubocop, we've added some changes to the rubocop yml file which is the reason we are seeing these formatting diffs now.